### PR TITLE
DXE-903 Fix Image and Video Manager docs

### PR DESCRIPTION
**Pull request by dstopka**

### CODE REVIEW

**Review by lkowalsk-akamai-com — APPROVED** [review]

**Review by mgwoj — APPROVED** [review]

File: docs/resources/imaging_policy_set.md [hunk 1]
```diff
@@ -28,5 +28,5 @@ resource "akamai_imaging_policy_set" "example_policy_set" {
 This data source supports these arguments:
```
Comment (robertolopezlopez) [inline docs/resources/imaging_policy_set.md]:
```suggestion
This resource supports these arguments:
```

**Review by robertolopezlopez — CHANGES_REQUESTED** [review]
These lines should be changed as well (Github just allows to suggestions some lines above/below your changes, and these are not): 
* https://github.com/akamai/terraform-provider-akamai/pull/304/files#diff-e20d3d9ad93a0a90a00b6f0ed2d25efc142c3ee8310040c58a1926630780c272R11
* https://github.com/akamai/terraform-provider-akamai/pull/304/files#diff-12b72681ab23aea941098a6ff07d64f401bace49e1a1e2860acefe53e0a222aeR44

File: docs/resources/imaging_policy_set.md (hunk 1 above)
Comment (robertolopezlopez) [reply docs/resources/imaging_policy_set.md]: Links are not working. Some screen shots: 
image
in imaging_policy_video.md

Comment (robertolopezlopez) [reply docs/resources/imaging_policy_set.md]: image
in imaging_policy_image.md

Comment (dstopka) [reply docs/resources/imaging_policy_set.md]: Thanks, adding it

**Review by robertolopezlopez — APPROVED** [review]

**Review by mgwoj — APPROVED** [review]

**Pull request merged**